workspaces/homepage/plugins/dynamic-home-page/src/components/EntitySection/TagList.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+
+import Box from '@mui/material/Box';
+import Chip from '@mui/material/Chip';
+import Typography from '@mui/material/Typography';
+import { KINDS } from '../../utils/constants';
+
+type KindKeys = keyof typeof KINDS;
+
+interface TagListProps {
+  tags: string[];
+  kind: string;
+}
+
+const TagList: React.FC<TagListProps> = ({ tags, kind }) => {
+  const hiddenCount = tags.length - 3;
+  return (
+    <Box
+      sx={{
+        height: '72px',
+        overflow: 'hidden',
+        display: 'flex',
+        flexWrap: 'wrap',
+        alignItems: 'flex-end',
+        gap: 0.5,
+      }}
+    >
+      <Chip
+        key={kind}
+        label={
+          <Typography sx={{ fontSize: '0.8rem', fontWeight: 400 }}>
+            {kind}
+          </Typography>
+        }
+        sx={{
+          backgroundColor: KINDS[kind.toLocaleUpperCase() as KindKeys]?.fill,
+        }}
```

Comment:
Can you set also a (text) color when we use really specific background color here. Maybe simply black?

It looks okaysh on the light theme

![Screenshot From 2025-05-07 11-09-12](https://github.com/user-attachments/assets/2e51aa4d-a738-4a68-bd35-c4836fbc5937)

but on dark theme its difficult to read

![Screenshot From 2025-05-07 11-09-03](https://github.com/user-attachments/assets/de81b9b3-adce-4c4c-8a3d-eff43f2589d1)